gists_database/importer.py

```diff
@@ -1,5 +1,46 @@
 import requests
+import sqlite3
+import os
+from os.path import dirname as dot
+
+BASE_PROJECT_PATH = dot(dot(__file__))
+DATABASE_SCHEMA_PATH = os.path.join(BASE_PROJECT_PATH, 'schema.sql')
+
+API_ENDPOINT = 'https://api.github.com'
+
+INSERT_QUERY = """
+    INSERT INTO gists 
+    (github_id, html_url, git_pull_url, git_push_url, commits_url, forks_url,
+     public, created_at, updated_at, comments, comments_url)
+    VALUES (:github_id, :html_url, :git_pull_url, :git_push_url, :commits_url,
+     :forks_url, :public, :created_at, :updated_at,:comments, :comments_url);
+"""
 
 
 def import_gists_to_database(db, username, commit=True):
-    pass
+    USER_GIST_URL = 'https://api.github.com/users/{username}/gists'.format(username=username)
+
+    r = requests.get(USER_GIST_URL)
+    r.raise_for_status()
+
+    r = r.json()
+
+    for data in r:
```

Comment:
I think it's better just to say `for data in r.json()` than renaming the r variable for something as simple. Personal opinion though. Also, I'd say `for gist in r.json()`.

Reply:
I completely agree with the first suggestion.  I think an earlier implementation I had them split for a reason but it does look stupid now.  

I went with data instead of gist to clarify that they weren't `Gist` objects and just fields.